src/integration-tests/mentees/mentees.spec.js

```diff
@@ -6,38 +6,32 @@
  */
 
 const commonHelper = require('@commonTests')
-const sessionsData = require('../sessions/sessionsData')
 const schema = require('./responseSchema')
-
+let loginInfo
 describe('mentoring/v1/mentees', function () {
 	beforeAll(async () => {
-		await commonHelper.logIn()
-	})
-	it('/joinSession', async () => {
-		let sessionId = await sessionsData.insertSession((now = true), (sessionStatus = 'live'), (meetingInfo = true))
-		await sessionsData.insertSessionAttendee(sessionId)
-		let res = await request.post('/mentoring/v1/mentees/joinSession/' + sessionId)
-		expect(res.statusCode).toBe(200)
-		expect(res.body).toMatchSchema(schema.joinSessionSchema)
-	})
-	it('/homeFeed', async () => {
-		let res = await request.get('/mentoring/v1/mentees/homeFeed')
-		expect(res.statusCode).toBe(200)
-		expect(res.body).toMatchSchema(schema.homeFeedSchema)
-	})
-	it('/reports', async () => {
-		let res = await request.get('/mentoring/v1/mentees/reports').query({ filterType: 'QUARTERLY' })
-		expect(res.statusCode).toBe(200)
-		expect(res.body).toMatchSchema(schema.reportsSchema)
-	})
-	it('/profile', async () => {
-		let res = await request.get('/mentoring/v1/mentees/profile')
+		loginInfo = await commonHelper.logIn()
+		await request.post('/mentoring/v1/profile/create').send({
+			designation: ['beo', 'deo', 'testt'],
+			area_of_expertise: ['educational_leadership', 'sqaa'],
+			education_qualification: 'MBA',
+			tags: ['Experienced', 'Technical'],
+			visibility: 'visible',
+			organisation_ids: [1],
+			external_session_visibility: 'limited',
```

Comment:
@adithyadinesh0412 currently defined policied are CURRENT< ASSOCIATED and ALL. Please use that